docs/bug-artifacts.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Filing a Bug with the SDK
+---
+
+If you encounter a bug with the Tableau Connector SDk, please file it as a github issue. We will take a look at it and respond as soon as we are able.
+
+# Artifacts
+When filing an bug on github, attaching the following to the issue:
+- A readme file that contains the following (can be in the issue itself):
+    - Your name and company
+    - A short description of the bug
+    - Repro steps with screenshots
+    - The product(s) it appears in
+    - The exact version of the Tableau products you are using (ie Desktop 2020.3.1, Server 2020.2.2, TDVT 2.1.1, etc), as relevant to the bug
```

Comment:
TDVT and Desktop version are technically independent I believe.  Its probably worth clarifying we need both in the TDVT case.